tests/svelte.ts

```diff
@@ -16,6 +16,7 @@ export async function test(options: RunOptions) {
 		repo: 'sveltejs/kit',
 		branch: 'master',
 		overrides: {
+			svelte: 'latest',
```

Comment:
Either override to version in v-p-s pkg or override for both v-p-s and kit

Reply:
Oh uh I somehow missed the other `runInRepo` function 😅